development/contribute-to-docs.md

```diff
@@ -0,0 +1,146 @@
+ability to use your contributions. You will be prompted to sign the
+CLA during the pull request process.
+
+# Making minor changes
+If you want to make only minor changes to docs, you can make corrections 
+and submit pull requests on the GitHub website. Go to the file you want to 
+correct and click the 'pencil' icon to edit. Once done, GitHub gives you 
+an option to submit a pull request at the bottom of the page.
+
+# Clone the documentation repository
+Timescale documentation is in the `docs.timescale.com-content` 
+[repository][timescale-docs-github]. You can clone this repository from 
+the command line like so:
+
+```bash
+git clone git@github.com:timescale/docs.timescale.com-content.git
+```
+
+Outside contributors will have to push their changes to a branch. You 
+will need to fork the repository, then pull from your fork, make changes to 
+a branch, then push changes to a branch on your fork. From there, you can 
+submit a pull request. See the GitHub instructions on 
+[collaborating with pull requests][github-collaborating-instructions] for 
+more details.
+
+Once cloned, you should be able to open the entire Timescale 
+documentation repository in a code editor of your choice.
+
+You can create a new branch in git as well:
+
+```bash
+git checkout -b my-branch-name
+```
+
+# Modifying a file
+
+Once you’re in your new branch you can start editing your doc as you normally would.
+
+At any time, you can see which files have been edited with the following git command:
+
+```bash
+git status
+```
+
+# Adding a new item to the page structure
+
+Often, you may want to add a new file to Timescale docs. To do so, you must 
+edit the `page-index/page-index.js` file. You should be able to glean 
+which lines to add and how to modify them based on the existing structure of 
+the file. If you need help, let us know in the 
+[Timescale community Slack][timescale-slack].
+
+# Submitting a pull request
+
+A pull request is a method of submitting contributions to an open development 
+project. It occurs when a developer asks for changes committed to an external 
+repository to be considered for inclusion in a project's main repository after 
+the peer review.
+
+Once you’ve completed all of your edits and are ready to submit a pull request, 
+you will first need to add all your edited files to your branch:
+
+```bash
+git add .
+```
+
+Now, you need to commit your changes along with a message explaining what changes 
+you’ve made to the repository.
+
+```bash
+git commit -m “a very brief explanation of what your pull request contains”
+```
+
+Then you have to upload your changes to GitHub:
+
+```bash
+git push origin my-branch-name
+```
+
+You will then need to complete the process on the 
+[Timescale Docs GitHub repository][timescale-docs-github]. Click on 
+the `New Pull Request` button and follow the instructions to create a 
+pull request using the branch you uploaded.
+
```

Comment:
Clarify that if you have multiple commits, they should be "squashed" before being committed.

(I wonder if we should also tell people to "rebase" their commits before merging, or whether we are comfortable with github doing rebasing these days?  cc: @solugebefola ?  That's obviously a bit advanced...)

Reply:
Ah, you address the rebase below.  👍